pymc_experimental/distributions/continuous.py

```diff
@@ -216,3 +226,62 @@ def moment(rv, size, mu, sigma, xi):
         if not rv_size_is_none(size):
             mode = pt.full(size, mode)
         return mode
+
+
+class PCPriorStudentT_dof_RV(RandomVariable):
```

Comment:
needs a docstring

Reply:
Usually we don't document the RV, but the Distribution class, which doesn't have a docstring either